Makefile

```diff
@@ -25,7 +25,7 @@ build-cross:
 
 .PHONY: test
 test:
-	go test -timeout 3m ${TESTFLAGS} -v ./...
+	$(GO) test -timeout 3m ${TESTFLAGS} -v ./...
```

Comment:
I'm not sure if overriding versions still work with their recent changes to automatically install versions and downgrade language versions based on go.mod 🤔

See https://tip.golang.org/doc/toolchain